### ISSUE TITLE
ArithmeticException: / by zero

### ISSUE THREAD

**Issue by Dmitchell94** [gigaherz/Ender-Rift#5]
Hi,

Noticed this crash report and thought i should notify you of the crash.
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  CorePlugin (Fluxed Redstone-1.0.1.9.jar)
  TransformerLoader (OpenComputers-MC1.8.9-1.5.21.1.jar)
  LoadingPlugin (Bloodmoon-MC1.8.8-1.4.jar)
  FldLoadingPlugin (FastLeafDecay-1.8-1.6.jar)
  LoadingPlugin (RandomThings-MC1.8.9-3.5.8.jar)
  ThaumcraftLoader (Thaumcraft-1.8.9-5.1.3.jar)
  thaumicinfusion (ThaumicInfusion-4.75.jar)
  LoadingPlugin (HardcoreDarkness-MC1.8.8-1.8.jar)
  NowWithRendering (redstonepaste-mc1.8.8-1.7.2.jar)
  CCLCorePlugin (CodeChickenLib-1.8-1.1.2.139-universal.jar)
Contact their authors BEFORE contacting forge

// Quite honestly, I wouldn't worry myself about that.

Time: 1/23/16 9:28 AM
Description: Ticking block entity

java.lang.ArithmeticException: / by zero
	at gigaherz.enderRift.blocks.TileGenerator.func_73660_a(TileGenerator.java:120)
	at net.minecraft.world.World.func_72939_s(World.java:1716)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:544)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:701)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:344)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:605)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:481)
	at java.lang.Thread.run(Unknown Source)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at gigaherz.enderRift.blocks.TileGenerator.func_73660_a(TileGenerator.java:120)

-- Block entity being ticked --
Details:
	Name: tileGenerator // gigaherz.enderRift.blocks.TileGenerator
	Block type: ID #410 (tile.enderrift.blockGenerator // gigaherz.enderRift.blocks.BlockGenerator)
	Block data value: 2 / 0x2 / 0b0010
	Block location: World: (4,59,118), Chunk: (at 4,3,6 in 0,7; contains blocks 0,0,112 to 15,255,127), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Actual block type: ID #410 (tile.enderrift.blockGenerator // gigaherz.enderRift.blocks.BlockGenerator)
	Actual block data value: 2 / 0x2 / 0b0010
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1716)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:544)

-- Affected level --
Details:
	Level name: world
	All players: 0 total; []
	Chunk stats: ServerChunkCache: 625 Drop: 0
	Level seed: -898271994946802148
	Level generator: ID 00 - default, ver 1. Features enabled: true
	Level generator options: 
	Level spawn location: 44.00,64.00,256.00 - World: (44,64,256), Chunk: (at 12,4,0 in 2,16; contains blocks 32,0,256 to 47,255,271), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 1807143 game time, 1825183 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 24520 (now: false), thunder time: 89533 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:701)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:344)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:605)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:481)
	at java.lang.Thread.run(Unknown Source)

-- System Details --
Details:
	Minecraft Version: 1.8.9
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_66, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1543948064 bytes (1472 MB) / 2520252416 bytes (2403 MB) up to 4772593664 bytes (4551 MB)
	JVM Flags: 2 total; -Xms1024M -Xmx5G
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP 9.19 Powered by Forge 11.15.0.1710 49 mods loaded, 49 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.18} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.8.9-11.15.0.1710-universal.jar) 
	UCHIJAAAA	Forge{11.15.0.1710} [Minecraft Forge] (forge-1.8.9-11.15.0.1710-universal.jar) 
	UCHIJAAAA	OpenComputers|Core{1.5.21.1} [OpenComputers (Core)] (minecraft.jar) 
	UCHIJAAAA	securitycraft{v1.8.1} [SecurityCraft] ([1.8.8] SecurityCraft v1.8.1.jar) 
	UCHIJAAAA	additionalbanners{1.0.0} [Additional Banners] (AdditionalBanners-1.1.0.jar) 
	UCHIJAAAA	bagginses{2.1.4} [Bagginses] (Bagginses-1.8.9-2.1.4.jar) 
	UCHIJAAAA	BiomesOPlenty{3.0.0} [Biomes O' Plenty] (BiomesOPlenty-1.8.9-3.0.0.1673-universal.jar) 
	UCHIJAAAA	BloodMagic{1.8.9-2.0.0-6} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.8.9-2.0.0-6.jar) 
	UCHIJAAAA	Bloodmoon{1.2} [Bloodmoon] (Bloodmoon-MC1.8.8-1.4.jar) 
	UCHIJAAAA	bookshelfapi{2.2.3} [Bookshelf] (Bookshelf-1.8.9-2.2.3.jar) 
	UCHIJAAAA	ChestTransporter{2.2.0} [Chest Transporter] (ChestTransporter-2.2.0-MC1.8.9.jar) 
	UCHIJAAAA	ComputerCraft{1.78} [ComputerCraft] (ComputerCraft1.78.jar) 
	UCHIJAAAA	enderrift{0.70} [Ender-Rift] (EnderRift-0.70.jar) 
	UCHIJAAAA	EnderZoo{1.8.9-1.1.0} [Ender Zoo] (EnderZoo-1.8.9-1.1.2.jar) 
	UCHIJAAAA	Baubles{1.1.3.0} [Baubles] (Baubles-1.8.9-1.1.3.0.jar) 
	UCHIJAAAA	Thaumcraft{5.1.3} [Thaumcraft] (Thaumcraft-1.8.9-5.1.3.jar) 
	UCHIJAAAA	ExtraFood{0.7.51} [Extra Food] (ExtraFood-1.8.9-0.7.51.jar) 
	UCHIJAAAA	fastleafdecay{1.6} [Fast Leaf Decay] (FastLeafDecay-1.8-1.6.jar) 
	UCHIJAAAA	mcmultipart{1.0.5} [MCMultiPart] (MCMultiPart-1.0.5-universal.jar) 
	UCHIJAAAA	reborncore{1.3.1.9} [RebornCore] (RebornCore-1.8.9-1.3.1.9-universal.jar) 
	UCHIJAAAA	fluxedredstone{1.0.1.9} [FluxedRedstone] (Fluxed Redstone-1.0.1.9.jar) 
	UCHIJAAAA	fluxedsolars{$GRADLEVERSION} [Fluxed Solars] (fluxedsolars-1.4.jar) 
	UCHIJAAAA	foundry{1.3.3.1} [Foundry] (Foundry-1.8.9-1.3.3.1.jar) 
	UCHIJAAAA	FunOres{1.0.4} [Fun Ores] (FunOres-1.8.9-1.0.4-25.jar) 
	UCHIJAAAA	gravestone{1.1.2} [Gravestone] (Gravestone Mod 1.1.2.jar) 
	UCHIJAAAA	HardcoreDarkness{1.8} [Hardcore Darkness] (HardcoreDarkness-MC1.8.8-1.8.jar) 
	UCHIJAAAA	IGWMod{1.2.3-3} [In-Game wiki Mod] (IGW-Mod-1.8.9-1.2.3-3-universal.jar) 
	UCHIJAAAA	IronChest{6.0.121.768} [Iron Chest] (ironchest-1.8.9-6.0.121.768.jar) 
	UCHIJAAAA	JEI{2.21.0.98} [Just Enough Items] (jei_1.8.9-2.21.0.98.jar) 
	UCHIJAAAA	MoreFurnaces{1.5.0} [More Furnaces] (MoreFurnaces-1.5.0-MC1.8.9.jar) 
	UCHIJAAAA	MultiPageChest{1.4.0} [Multi Page Chest] (MultiPageChest-1.4.0-MC1.8.9.jar) 
	UCHIJAAAA	neotech{2.2.3} [NeoTech] (NeoTech-1.8.9-2.2.3.jar) 
	UCHIJAAAA	OpenComputers{1.5.21.1} [OpenComputers] (OpenComputers-MC1.8.9-1.5.21.1.jar) 
	UCHIJAAAA	PneumaticCraft{1.13.2-3} [PneumaticCraft] (PneumaticCraft-1.8.9-1.13.2-3-universal.jar) 
	UCHIJAAAA	RandomThings{3.5.8} [Random Things] (RandomThings-MC1.8.9-3.5.8.jar) 
	UCHIJAAAA	redstonepaste{1.7.2} [Redstone Paste] (redstonepaste-mc1.8.8-1.7.2.jar) 
	UCHIJAAAA	Roguelike{1.4.5} [Roguelike Dungeons] (RoguelikeDungeons-1.8.9-1.4.5.jar) 
	UCHIJAAAA	sem{1.4.3} [Steam Engines Mod] (sem-mc1_8_9-1.4.3.jar) 
	UCHIJAAAA	semmagie{1.4.3} [Steam Engines Mod - Magie Addon] (sem-mc1_8_9-1.4.3.jar) 
	UCHIJAAAA	sodacanutils{2.0.0} [Sodacan Utilities] (sodacanutils-2.0.3.jar) 
	UCHIJAAAA	Talismans 2{2.2.0} [Talismans 2] (Talismans2-1.8.9-2.2.0.jar) 
	UCHIJAAAA	thaumicinfusion{4.75} [Thaumic Infusion] (ThaumicInfusion-4.75.jar) 
	UCHIJAAAA	thelittlethings{1.2.1-Forge1.8.9} [The Little Things] (The Little Things-v.1.2.1-mc1.8.9.jar) 
	UCHIJAAAA	thutcore{2.2.1} [ThutCore] (thutcore-2.2.1.jar) 
	UCHIJAAAA	thuttech{2.2.5} [Thut's Tech] (thuttech-2.2.5.jar) 
	UCHIJAAAA	VeinMiner{0.31.2_build.unknown} [Vein Miner] (VeinMiner-1.8.9_0.31.2.unknown.jar) 
	UCHIJAAAA	VeinMinerModSupport{0.31.2_build.unknown} [Mod Support] (VeinMiner-1.8.9_0.31.2.unknown.jar) 
	UCHIJAAAA	worldedit{6.1.1} [WorldEdit] (worldedit-forge-mc1.8.9-6.1.1.jar) 
	Loaded coremods (and transformers): 
CorePlugin (Fluxed Redstone-1.0.1.9.jar)
  
TransformerLoader (OpenComputers-MC1.8.9-1.5.21.1.jar)
  li.cil.oc.common.asm.ClassTransformer
LoadingPlugin (Bloodmoon-MC1.8.8-1.4.jar)
  lumien.bloodmoon.asm.ClassTransformer
FldLoadingPlugin (FastLeafDecay-1.8-1.6.jar)
  com.olafski.fastleafdecay.FldTransformer
LoadingPlugin (RandomThings-MC1.8.9-3.5.8.jar)
  lumien.randomthings.asm.ClassTransformer
ThaumcraftLoader (Thaumcraft-1.8.9-5.1.3.jar)
  
thaumicinfusion (ThaumicInfusion-4.75.jar)
  drunkmafia.thaumicinfusion.common.asm.BlockTransformer
  drunkmafia.thaumicinfusion.common.asm.WorldTransformer
LoadingPlugin (HardcoreDarkness-MC1.8.8-1.8.jar)
  lumien.hardcoredarkness.asm.ClassTransformer
NowWithRendering (redstonepaste-mc1.8.8-1.7.2.jar)
  net.fybertech.nwr.NowWithRendering
CCLCorePlugin (CodeChickenLib-1.8-1.1.2.139-universal.jar)
  codechicken.lib.asm.ClassHeirachyManager
  codechicken.lib.asm.RenderHookTransformer
	Profiler Position: N/A (disabled)
	Player Count: 0 / 20; []
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)
```
Seems that for some reason you divide by zero

**Comment by gigaherz** [comment]
Thanks for the report, I'll look into it right now.

**Comment by gigaherz** [comment]
The issue should be fixed in [v0.71](https://github.com/gigaherz/Ender-Rift/releases/tag/v0.71).
